### PR TITLE
Update _centralized_login.md

### PR DESCRIPTION
**Pull request by alexisluque**
* Remove unused import.
* Add missing imports.
* Improve instructions for creating `AuthService`.

### CODE REVIEW

**Comment by damieng** [comment]
Can you put a description of the purpose of the change in the PR please?  It helps us evaluate pull requests for subsequent merging.

File: articles/quickstart/spa/angular2/_includes/_centralized_login.md [hunk 1]
```diff
@@ -87,7 +86,6 @@ export class AuthService {
   public handleAuthentication(): void {
     this.auth0.parseHash((err, authResult) => {
       if (authResult && authResult.accessToken && authResult.idToken) {
-        window.location.hash = '';
```
Comment (damieng) [inline articles/quickstart/spa/angular2/_includes/_centralized_login.md]: Typically this is done to remove the authentication hash from the page history.  Is there a reason it is removed here?

Comment (alexisluque) [reply articles/quickstart/spa/angular2/_includes/_centralized_login.md]: You're right, this line shouldn't be removed. When removing the authentication hash and using Firefox, the app get stuck in `http://localhost:3000/callback`. In Chrome and Opera the app works as expected.

**Review by damieng — APPROVED** [review]

**Review by damieng — CHANGES_REQUESTED** [review]
Please add back window.location (line 90)

**Review by damieng — APPROVED** [review]

**Pull request merged**